int16 xcmean
int16 ycmean
float32 xmean
float32 ymean
float32 xgmean
float32 ygmean
float32 xvari
float32 yvari
float32 angle
float32 xdis
float32 vdis
float32 vydis
float32 dis_x
float32 dis_y
float32 dis_xg
float32 dis_yg
float32 vxabs
float32 vyabs
float32 vabs
time timestamp
float32 heading